setup/system.sh

```diff
@@ -11,13 +11,12 @@ source setup/functions.sh # load our functions
 #
 # First set the hostname in the configuration file, then activate the setting
 
-echo $PRIMARY_HOSTNAME > /etc/hostname
-hostname $PRIMARY_HOSTNAME
+hostnamectl set-hostname "$PRIMARY_HOSTNAME"
```

Comment:
Not that we're worried about it for long - but I don't think this will work on Ubuntu 14.04?

Reply:
Yes, it does. It actually works on almost every currently supported Linux distribution.

Reply:
The `hostnamectl` command is a part of systemd. I hadn't realised that, despite mostly using upstart, systemd-services are included in 14.04. I'm not certain if *all* 14.04 releases have it - but I don't think we'd be concerned, as long as it is in the latest one.

Good enough for me. :)